google-cloud-clients/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -67,6 +70,7 @@
       DateTimeFormatter.ofPattern("HH:mm:ss.SSSSSS");
   private static final DateTimeFormatter datetimeFormatter =
       DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSS");
+  private static final DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
Please use `DateTimeFormatter` instead of the java standard format.

Reply:
DateTimeFormatter is only available after java 8.

Reply:
Please see line 72.

Reply:
Updated to remove formatter and replaced with utility method from google cloud date. Sorry I misread your comment earlier and though of jdk8s datetimeformatter.